content/en/docs/reference/command-line-tools-reference/kube-controller-manager.md

```diff
@@ -208,7 +208,7 @@ kube-controller-manager [flags]
 <td colspan="2">--cluster-signing-duration duration&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;Default: 8760h0m0s</td>
 </tr>
 <tr>
-<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>The length of duration signed certificates will be given.</p></td>
+<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>The max length of duration signed certificates will be given.  Individual CSRs may request shorter certs by setting spec.expirationSeconds.</p></td>
```

Comment:
```suggestion
<td></td><td style="line-height: 130%; word-wrap: break-word;"><p>The max length of duration signed certificates will be given.  Individual CSRs may request shorter certs by setting `spec.expirationSeconds`.</p></td>
```

Reply:
We don't (yet) have a good way to put Markdown or other formatting in the source code that these are generated from.